src/main/java/seedu/address/logic/parser/AddProjectCommandParser.java

```diff
@@ -35,6 +35,7 @@ public AddProjectCommand parse(String args) throws ParseException {
         }
 
         argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_PROJECT);
+        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_EMPLOYEE);
```

Comment:
Lines 37 to 38 can be shortened to: `argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_PROJECT, PREFIX_EMPLOYEE);`